packages/form-system/stories/01-overview.stories.mdx

```diff
@@ -0,0 +1,101 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { render } from '@lion/core';
+import '@lion/input/lion-input.js';
+import { Validator } from '@lion/validate';
+import './helper-wc/h-output.js';
+
+<Meta title="Form Fundaments/Overview" />
+
+# Form System Overview
+
+This page should be used as a starting point when first using the Form System.
+It provides an overview of its essential building blocks and provides links to detailed explanations
+of most of its core concepts.
+
+## Building Blocks
+
+Our Form System is built from a set of very fundamental building blocks: `form control`s, `field`s and `fieldset`s.
+
+### Form Controls
+
+`Form control`s are the most fundamental building block of our Form System.
+They are the fundament of both `field`s, and `fieldset`s and provide a normalized, predictable api
```

Comment:
```suggestion
They are the base of both `fields`, and `fieldsets` and provide a normalized, predictable interface
```

or 

```suggestion
They are at the core of both `fields`, and `fieldsets` and provide a normalized, predictable interface
```

Reply:
All terms above are interchangeable to me; feel free to update ;)